gnomad_hail/utils/annotations.py

```diff
@@ -419,3 +419,63 @@ def annotate_freq(
 
     # Return MT with freq row annotation
     return mt.annotate_rows(freq=freq_expr).drop('_freq_meta')
+
+
+def get_annotations_hists(
+    ht: hl.Table, annotations_hists: Dict[str, Tuple],
```

Comment:
I think moving `annotations_hists` to the next line would increase readability

Reply:
Plug for [Black](https://pypi.org/project/black/) to automatically resolve all formatting discussions.